viewing_party/party.py

```diff
@@ -1,23 +1,175 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {}
+    if title is None:
+        return None
+    if genre is None:
+        return None
+    if rating is None:
+        return None
+    
+    movie_dict["title"] = title
+    movie_dict["genre"] = genre
+    movie_dict["rating"] = rating
+
+    return movie_dict

```

Comment:
A few points here:
- The more pythonic way of checking that a value is `None` would be to check that the value is falsey like `if not title`.
- We can combine these 3 checks all into one line so the function is more concise. 
- We can directly return a dictionary literal without creating `movie_dict` first

```suggestion
    if not (title and genre and rating):
        return None
        
    return {
        "title": title, 
        "genre": genre, 
        "rating": rating 
    } 
```